hail/python/test/hail/expr/test_ndarrays.py

```diff
@@ -14,8 +14,9 @@ def assert_ndarrays(asserter, exprs_and_expecteds):
     expr_tuple = hl.tuple(exprs)
     evaled_exprs = hl.eval(expr_tuple)
 
-    for (evaled, expected) in zip(evaled_exprs, expecteds):
-        assert asserter(evaled, expected)
+    evaled_and_expected = zip(evaled_exprs, expecteds)
+    for (idx, (evaled, expected)) in enumerate(evaled_and_expected):
+        assert asserter(evaled, expected), f"NDArray comparison {idx} failed"
```

Comment:
Hmm. Should we include `evaled` and `expected` in the error message?

Reply:
pytest will probably give you that already, actually.

Reply:
Pytest does evaled and expected automatically, but since I was running tests in groups it was getting confusing which one failed. This strictly adds information